src/components/widget/Lookup/RawLookup.js

```diff
@@ -141,6 +141,12 @@ export class RawLookup extends Component {
     });
   };
 
+  handleDropdownListToggle = (val, mouse) => {
+    const { field, onDropdownListToggle } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['field' is missing in props validation (react/prop-types)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4689744)